### PR TITLE
Minor changes on PTBR language

### PR DESCRIPTION
**Pull request by bitcoinuser**

### CODE REVIEW

**Comment by fiatjaf** [comment]
In my understanding all the changes here are improvements that increase overall clarity.

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 1]
```diff
@@ -398,12 +398,12 @@
 
   <string name="lnurl_withdraw_amount_label">Valor a receber</string>
   <string name="lnurl_withdraw_description_label">Detalhes</string>
-  <string name="lnurl_withdraw_service_host_label"><![CDATA[Retirar fundos de<br/><b>%1$s</b>.]]></string>
-  <string name="lnurl_withdraw_confirm_button">Retirar</string>
+  <string name="lnurl_withdraw_service_host_label"><![CDATA[Receber fundos de<br/><b>%1$s</b>.]]></string>
+  <string name="lnurl_withdraw_confirm_button">Confirmar</string>
   <string name="lnurl_withdraw_wait">Solicitando valores…</string>
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:403]:
```suggestion
  <string name="lnurl_withdraw_wait">Recebendo valores…</string>
```

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 2]
```diff
@@ -398,12 +398,12 @@
...
   <string name="lnurl_withdraw_description_label">Detalhes</string>
-  <string name="lnurl_withdraw_service_host_label"><![CDATA[Retirar fundos de<br/><b>%1$s</b>.]]></string>
-  <string name="lnurl_withdraw_confirm_button">Retirar</string>
-  <string name="lnurl_withdraw_wait">Solicitando valores…</string>
-  <string name="lnurl_withdraw_success">Aguarde enquanto %1$s está enviando os valores.\n\nMantenha o aplicativo online.</string>
-  <string name="lnurl_withdraw_error_internal">Não foi possível processar esta retirada. Tente novamente.</string>
-  <string name="lnurl_withdraw_error_invalid">Esta solicitação de retirada é inválida. Tente novamente.</string>
+  <string name="lnurl_withdraw_service_host_label"><![CDATA[Receber fundos de<br/><b>%1$s</b>.]]></string>
+  <string name="lnurl_withdraw_confirm_button">Confirmar</string>
+  <string name="lnurl_withdraw_wait">Recebendo valores…</string>
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:403]:
```suggestion
  <string name="lnurl_withdraw_wait">Resgatando valores…</string>
```

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 3]
```diff
@@ -398,12 +398,12 @@
...
-  <string name="lnurl_withdraw_wait">Solicitando valores…</string>
-  <string name="lnurl_withdraw_success">Aguarde enquanto %1$s está enviando os valores.\n\nMantenha o aplicativo online.</string>
-  <string name="lnurl_withdraw_error_internal">Não foi possível processar esta retirada. Tente novamente.</string>
-  <string name="lnurl_withdraw_error_invalid">Esta solicitação de retirada é inválida. Tente novamente.</string>
+  <string name="lnurl_withdraw_service_host_label"><![CDATA[Receber fundos de<br/><b>%1$s</b>.]]></string>
+  <string name="lnurl_withdraw_confirm_button">Confirmar</string>
+  <string name="lnurl_withdraw_wait">Resgatando valores…</string>
+  <string name="lnurl_withdraw_success">Aguarde enquanto %1$s está enviando os valores.\n\nMantenha o aplicativo aberto.</string>
+  <string name="lnurl_withdraw_error_internal">Não foi possível receber este pagamento. Tente novamente.</string>
+  <string name="lnurl_withdraw_error_invalid">Esta solicitação de pagamento é inválida. Tente novamente.</string>
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:406]:
```suggestion
  <string name="lnurl_withdraw_error_invalid">Esta solicitação para resgatar pagamento é inválida. Tente novamente.</string>
```

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 4]
```diff
@@ -398,12 +398,12 @@
...
-  <string name="lnurl_withdraw_confirm_button">Retirar</string>
-  <string name="lnurl_withdraw_wait">Solicitando valores…</string>
-  <string name="lnurl_withdraw_success">Aguarde enquanto %1$s está enviando os valores.\n\nMantenha o aplicativo online.</string>
-  <string name="lnurl_withdraw_error_internal">Não foi possível processar esta retirada. Tente novamente.</string>
-  <string name="lnurl_withdraw_error_invalid">Esta solicitação de retirada é inválida. Tente novamente.</string>
+  <string name="lnurl_withdraw_service_host_label"><![CDATA[Receber fundos de<br/><b>%1$s</b>.]]></string>
+  <string name="lnurl_withdraw_confirm_button">Confirmar</string>
+  <string name="lnurl_withdraw_wait">Resgatando valores…</string>
+  <string name="lnurl_withdraw_success">Aguarde enquanto %1$s está enviando os valores.\n\nMantenha o aplicativo aberto.</string>
+  <string name="lnurl_withdraw_error_internal">Não foi possível receber este pagamento. Tente novamente.</string>
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:405]:
```suggestion
  <string name="lnurl_withdraw_error_internal">Não foi possível resgatar este pagamento. Tente novamente.</string>
```

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 5]
```diff
@@ -398,12 +398,12 @@
 
   <string name="lnurl_withdraw_amount_label">Valor a receber</string>
   <string name="lnurl_withdraw_description_label">Detalhes</string>
-  <string name="lnurl_withdraw_service_host_label"><![CDATA[Retirar fundos de<br/><b>%1$s</b>.]]></string>
-  <string name="lnurl_withdraw_confirm_button">Retirar</string>
-  <string name="lnurl_withdraw_wait">Solicitando valores…</string>
-  <string name="lnurl_withdraw_success">Aguarde enquanto %1$s está enviando os valores.\n\nMantenha o aplicativo online.</string>
-  <string name="lnurl_withdraw_error_internal">Não foi possível processar esta retirada. Tente novamente.</string>
-  <string name="lnurl_withdraw_error_invalid">Esta solicitação de retirada é inválida. Tente novamente.</string>
+  <string name="lnurl_withdraw_service_host_label"><![CDATA[Receber fundos de<br/><b>%1$s</b>.]]></string>
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:401]:
```suggestion
  <string name="lnurl_withdraw_service_host_label"><![CDATA[Resgatando fundos de<br/><b>%1$s</b>.]]></string>
```

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 6]
```diff
@@ -398,12 +398,12 @@
...
-  <string name="lnurl_withdraw_wait">Solicitando valores…</string>
-  <string name="lnurl_withdraw_success">Aguarde enquanto %1$s está enviando os valores.\n\nMantenha o aplicativo online.</string>
-  <string name="lnurl_withdraw_error_internal">Não foi possível processar esta retirada. Tente novamente.</string>
-  <string name="lnurl_withdraw_error_invalid">Esta solicitação de retirada é inválida. Tente novamente.</string>
+  <string name="lnurl_withdraw_service_host_label"><![CDATA[Resgatando fundos de<br/><b>%1$s</b>.]]></string>
+  <string name="lnurl_withdraw_confirm_button">Confirmar</string>
+  <string name="lnurl_withdraw_wait">Resgatando valores…</string>
+  <string name="lnurl_withdraw_success">Aguarde enquanto %1$s está enviando os valores.\n\nMantenha o aplicativo aberto.</string>
+  <string name="lnurl_withdraw_error_internal">Não foi possível resgatar este pagamento. Tente novamente.</string>
+  <string name="lnurl_withdraw_error_invalid">Esta solicitação para resgatar pagamento é inválida. Tente novamente.</string>
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:406]:
```suggestion
  <string name="lnurl_withdraw_error_invalid">Esta solicitação para resgate de pagamento é inválida. Tente novamente.</string>
```

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 7]
```diff
@@ -398,12 +398,12 @@
 
   <string name="lnurl_withdraw_amount_label">Valor a receber</string>
   <string name="lnurl_withdraw_description_label">Detalhes</string>
-  <string name="lnurl_withdraw_service_host_label"><![CDATA[Retirar fundos de<br/><b>%1$s</b>.]]></string>
-  <string name="lnurl_withdraw_confirm_button">Retirar</string>
-  <string name="lnurl_withdraw_wait">Solicitando valores…</string>
-  <string name="lnurl_withdraw_success">Aguarde enquanto %1$s está enviando os valores.\n\nMantenha o aplicativo online.</string>
-  <string name="lnurl_withdraw_error_internal">Não foi possível processar esta retirada. Tente novamente.</string>
-  <string name="lnurl_withdraw_error_invalid">Esta solicitação de retirada é inválida. Tente novamente.</string>
+  <string name="lnurl_withdraw_service_host_label"><![CDATA[Resgatando fundos de<br/><b>%1$s</b>.]]></string>
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:401]:
```suggestion
  <string name="lnurl_withdraw_service_host_label"><![CDATA[Você está resgatando fundos de<br/><b>%1$s</b>.]]></string>
```

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 8]
```diff
@@ -398,12 +398,12 @@
...
   <string name="lnurl_withdraw_amount_label">Valor a receber</string>
   <string name="lnurl_withdraw_description_label">Detalhes</string>
-  <string name="lnurl_withdraw_service_host_label"><![CDATA[Retirar fundos de<br/><b>%1$s</b>.]]></string>
-  <string name="lnurl_withdraw_confirm_button">Retirar</string>
-  <string name="lnurl_withdraw_wait">Solicitando valores…</string>
-  <string name="lnurl_withdraw_success">Aguarde enquanto %1$s está enviando os valores.\n\nMantenha o aplicativo online.</string>
-  <string name="lnurl_withdraw_error_internal">Não foi possível processar esta retirada. Tente novamente.</string>
-  <string name="lnurl_withdraw_error_invalid">Esta solicitação de retirada é inválida. Tente novamente.</string>
+  <string name="lnurl_withdraw_service_host_label"><![CDATA[Você está resgatando fundos de<br/><b>%1$s</b>.]]></string>
+  <string name="lnurl_withdraw_confirm_button">Confirmar</string>
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:402]:
```suggestion
  <string name="lnurl_withdraw_confirm_button">Receber</string>
```

**Comment by dpad85** [comment]
Thanks for this update!

**Pull request merged**